packages/google_mlkit_commons/ios/Classes/MLKVisionImage+FlutterPlugin.m

```diff
@@ -6,9 +6,19 @@ @implementation MLKVisionImage(FlutterPlugin)
 + (MLKVisionImage *)visionImageFromData:(NSDictionary *)imageData {
     NSString *imageType = imageData[@"type"];
     if ([@"file" isEqualToString:imageType]) {
-        return [self filePathToVisionImage:imageData[@"path"]];
+        int rotation = [imageData[@"metadata"][@"rotation"] intValue];
```

Comment:
This is not needed. When the image is a file, camera lens is not used.
Also, check what it is happening inside `filePathToVisionImage` : https://github.com/flutter-ml/google_ml_kit_flutter/blob/develop/packages/google_mlkit_commons/ios/Classes/MLKVisionImage%2BFlutterPlugin.m#L20-L25

Reply:
Orientation is already set in that method for files.

Reply:
Sorry, I didn't noticed that. I already made a new commit with those changes.